### PR TITLE
Organization Structure Project

### PR DESCRIPTION
**Pull request by elinaldosoft**
Organization structure tests.

### CODE REVIEW

**Comment by elinaldosoft** [comment]
I signed it

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+ipython
```
Comment (nitzpo) [inline requirements.txt]: Why add a requirements file for this?
It is already included in the install_requires argument for setup() in setup.py.

Comment (elinaldosoft) [inline requirements.txt]: To configure the environment in development mode, But I believe the file name should be 
requirements_dev.txt. What do you think ?

Comment (datashaman) [inline requirements.txt]: `python setup.py develop`

Comment (elinaldosoft) [inline requirements.txt]: Nice, thanks

**Pull request closed without merging**